lib/netext/grpcext/stream.go

```diff
@@ -78,15 +78,15 @@ func (s *Stream) receive() (*dynamicpb.Message, error) {
 // {"x":6,"y":4}
 // rather than the desired:
 // {"x":6,"y":4,"z":0}
-func (s *Stream) convert(msg *dynamicpb.Message) (map[string]interface{}, error) {
+func (s *Stream) convert(msg *dynamicpb.Message) (interface{}, error) {
```

Comment:
nit: can we make this into a function taking the marshaller and reusing it in both places?